src/main/resources/templates/registration.html

```diff
@@ -54,11 +55,14 @@ <h3>Anmeldung</h3>
         </small>
     </p>
 <script>
-const elems = [];
-function checkPrivacy(elem) {
-    if (elem.checked) elems.push(elem); else elems.pop();
-    document.forms[0].submit.disabled = elems.length !== 2;
-}
+    const elems = [];
+    /*<![CDATA[*/
```

Comment:
What's that with the CDATA comment?
Something special? Something new? ES-2030?

Reply:
I had to initialize a javascript value with a template variable from Quarkus Qute. This results in a syntax error in the IDE. The CDATA block looks like the correct way to solve this problem:

https://stackoverflow.com/questions/25687816/setting-up-a-javascript-variable-from-spring-model-by-using-thymeleaf

Another solution could be parsing a JavaScript string (with the Qute template variable) to an integer:

const numberOfRequiredElements = parseInt("{form.showVideoRecording ? 2 : 1}");